pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -513,7 +508,7 @@ let
           revert = true;
         })
       ]
-      ++ lib.optionals (chromiumVersionAtLeast "131" && stdenv.hostPlatform.isAarch64) [
+      ++ lib.optionals stdenv.hostPlatform.isAarch64 [
```

Comment:
Will upstream ever handle deal with whatever issue we're fixing here?
(I don't really know a lot about this, I have not read the discussion)

Reply:
This has since been fixed in both electron and chromium but was kept this way so far to spare us rebuilds. Its removal is already part of a bigger cleanup and refactor for which I have not opened a PR yet.